server.js

```diff
@@ -141,6 +156,54 @@ function postResponse(responseURL, response) {
   });
 }
 
+/**
+  Retrieve the bug with the given bug number using the REST api.
+  returns: an array [bugid, jsondata]
+*/
+function getBug(bugId) {
+  return httpinvoke('https://bugzilla.mozilla.org/rest/bug/' + bugId, 'GET').then(function(res) {
+    return [bugId, JSON.parse(res.body)];
+  });
+}
+
+/**
+  Send information about a specific bug to a channel via slack client
+  returns: null
+*/
+function respondToBug(bugId, channel, data) {
+  let response;
+
+  if (data.bugs && data.bugs.length) {
+    let bug = data.bugs[0];
+    response = `https://bugzil.la/${bug.id} — ${bug.resolution}, ${bug.assigned_to} — ${bug.summary}`;
```

Comment:
Did you want to replace the call to the readable bug statuses summarizer with this?